script.js

```diff
@@ -0,0 +1,12 @@
+function show() {
```

Comment:
- [ ] The menu is shown the first I open the website, kindly make sure that the menu is not shown until the user clicks on the Hamburger menu button

![Screenshot_20230801_090023](https://github.com/abdullah97-pro/Portfolio-Mobile-Project/assets/25227022/7996b450-4053-415f-be75-13aa1acde8f2)

Reply:
this is because of function name that I called in my script file. if i do not call function name so the script linters gives error